catalog/MDCDragons.xcodeproj/xcshareddata/xcschemes/MDCDragons.xcscheme

```diff
@@ -28,7 +28,7 @@
             buildForAnalyzing = "NO">
```

Comment:
is the changes in this file needed?

Reply:
It is needed for adding test target into scheme.